frontend/post/post.js

```diff
@@ -53,3 +53,11 @@ Post.prototype.remove = function remove(userName) {
     this.last_modified_by = userName;
     this.last_modified_date = new Date().toISOString();
 };
+
+
+Post.prototype.hasActivity = function hasActivity() {
+    var hasNoComments = this.number_of_comments === 0;
```

Comment:
is better to understand hasComments and hasLikes, when hasNoComments = false is not good to read, the same to !hasNoComments

Reply:
that's true